flowey/flowey_lib_hvlite/src/run_cargo_build.rs

```diff
@@ -482,3 +449,61 @@ impl FlowNode for Node {
         Ok(())
     }
 }
+
+/// Install dependencies and set environment variables necessary for cross
+/// compiling to another architecture.
+pub fn setup_cross_arch_build(
+    ctx: &mut NodeCtx<'_>,
```

Comment:
I don't suggest writing helper functions that take `ctx` like this. it makes diagnosing errors a lot harder, since you end up introducing "hidden" req deps to nodes.

in the future, I'd like to have a proc-macro (e.g: `#[flowey_helper_node]` or whatever) that lets you write this sort of function, but have it backed by a real standalone Node.

...in the meantime though, you'll want to make this its own node yourself.

you can just have it be an inline `mod` tho

Reply:
Ok I'll do that